photon-lib/src/main/java/org/photonvision/simulation/VideoSimUtil.java

```diff
@@ -100,69 +95,42 @@ public static Point[] getImageCorners(Size size) {
     }
 
     /**
-     * Gets the 8x8 (grayscale) image of a specific 16h5 AprilTag.
+     * Gets the 10x10 (grayscale) image of a specific 36h11 AprilTag.
      *
      * @param id The fiducial id of the desired tag
      */
-    public static Mat get16h5TagImage(int id) {
-        String name = kTag16h5ImageName;
-        String idString = String.valueOf(id);
-        name = name.substring(0, name.length() - idString.length()) + idString;
-
-        var resource = VideoSimUtil.class.getResource(kResourceTagImagesPath + name + ".png");
-
-        Mat result = new Mat();
-        // reading jar file
-        if (resource != null && resource.getPath().startsWith("file")) {
-            BufferedImage buf;
-            try {
-                buf = ImageIO.read(resource);
-            } catch (IOException e) {
-                System.err.println("Couldn't read tag image!");
-                return result;
-            }
-
-            result = new Mat(buf.getHeight(), buf.getWidth(), CvType.CV_8UC1);
-
-            byte[] px = new byte[1];
-            for (int y = 0; y < result.height(); y++) {
-                for (int x = 0; x < result.width(); x++) {
-                    px[0] = (byte) (buf.getRGB(x, y) & 0xFF);
-                    result.put(y, x, px);
-                }
-            }
-        }
-        // local IDE tests
-        else result = Imgcodecs.imread(kLocalTagImagesPath + name + ".png", Imgcodecs.IMREAD_GRAYSCALE);
+    public static Mat get36h11TagImage(int id) {
+        RawFrame frame = AprilTag.generate36h11AprilTagImage(id);
+        Mat result = new Mat(10, 10, CvType.CV_8UC1, frame.getData(), frame.getStride());
```

Comment:
I think that this mat constructor doesn't actually ever allocate data? Which means that if `frame` ever gets closed, we could end up with another use-after-free. Instead, make a clone of this mat and then close the RawFrame

Reply:
Should be fixed in the latest commit